pom.xml

```diff
@@ -98,7 +98,7 @@
     <commons-logging.version>1.2</commons-logging.version>
     <commons-csv.version>1.0</commons-csv.version>
     <sqlline.version>1.1.8</sqlline.version>
-    <guava.version>12.0.1</guava.version>
+    <guava.version>13.0.1</guava.version>
```

Comment:
Guava 13.0.1 seems to be required or else the tests fail with a class not found for com.google.common.util.concurrent.Service.Listener. HBase uses 12.0.1, will this cause issues?